src/main/java/seedu/address/logic/commands/DownloadSelectNotesCommand.java

```diff
@@ -84,8 +81,8 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
             if (fileSelect == null) {
                 availableDownloadFiles = getFileNames(driver);
                 driver.close();
-                return new CommandResult(Messages.MESSAGE_DOWNLOAD_SELECT_SUCCESS + moduleCode + "\r\n"
-                    + availableDownloadFiles);
+                return new CommandResult(Messages.MESSAGE_DOWNLOAD_SELECT_SUCCESS + moduleCode +
```

Comment:
Good job, however, it would still be clearer for the reader if your positive path is not hidden within an if statement.